packages/forms-web-app/src/views/pages/examination/select-file.njk

```diff
@@ -1,17 +1,197 @@
 {% extends "layouts/default.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+
+ {% block script %}
+	 <style>
+		 .dropZone.dragging::before {
+			 pointer-events: none;
+		 }
+	 </style>
+ {% endblock %}
 
 {% set backLinkUrl = backLinkUrl %}
 
 {% set pageTitle = pageTitle %}
 
 {% block pageTitle %}
-  {% if errors %}
-    {% set pageTitle = "Error: " + pageTitle %}
-  {% endif %}
-
-  {{ pageTitle }}
+	{% if errorSummary.length > 0 %}
+		{% set pageTitle = "Error: " + pageTitle %}
+	{% endif %}
+	{{ pageTitle }}
 {% endblock %}
 
 {% block content %}
-  <h1>{{ title }}</h1>
+	{% if errorSummary.length > 0 %}
+		{{ govukErrorSummary({
+			titleText: "There is a problem",
+			errorList: errorSummary
+		}) }}
+	{% endif %}
+	<div class="govuk-grid-row">
+		<div class="govuk-grid-column-two-thirds">
+			<span class="govuk-caption-l">{{ captionTitle }}</span>
+			<h1 class="govuk-heading-l">Upload files</h1>
+			<p class="govuk-body">Your file must be PDF, PNG, DOC, DOCX, JPG, JPEG, TIF, TIFF, XLS or XSLX. </p>
+			<p class="govuk-body">All files must be:</p>
+			<ul class="govuk-list govuk-list--bullet">
+				<li>a single document</li>
+				<li>smaller than {{ maxFileSizeInMb }}mb</li>
+			</ul>
+
+			<form name="fileUploadForm"
```

Comment:
Should this be it's own component ?

Reply:
If not lets lift the script and style out properly.

Reply:
Other teams are showing an interest in using this upload but perhaps future work.